components/map/basic/leafletLocalInstall.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs')
```

Comment:
File made to avoid problems on copy time of NON scss files and images.

Reply:
The thing is that leaflet doesn't have any SCSS file. Talking with @carlosvillu we concluded that this was the less painful solution. Modify .css dist file to .scss and delete the non used background-images to dismiss image load problems.

Reply:
I may be wrong but I disagree.
Background images will only be loaded if they are actually used in the DOM. But, if you add css statement to override waht you want you're done.

About the css file, you should be able to import css code into sass, right ??? 
http://sass-lang.com/documentation/file.SASS_CHANGELOG.html#CSS__import_Directives
All valid css is valid scss, by definition.